understanding/consistent-identification.html

```diff
@@ -58,185 +56,139 @@
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Actions are more predictable across pages.</dd>
+                  <dt>目標</dt>
+                  <dd>ページ間での動作をより予測可能にする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Identify repeating functions consistently.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>繰り返される機能を一貫して識別する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Consistently identified actions are especially important to people with disabilities.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>一貫して識別される動作は、障害のある人にとって特に重要である。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure consistent identification of functional
-                  components that appear repeatedly within a set of Web pages. A strategy that people
-                  who use screen readers use when operating a Web site is to rely heavily on their familiarity
-                  with functions that may appear on different Web pages. If identical functions have
-                  different labels (or, more generally, a different <a href="https://www.w3.org/TR/accname-aam-1.1/">accessible name</a>)
-                  on different Web pages, the site will be considerably more difficult
-                  to use. It may also be confusing and increase the cognitive load for people with cognitive
-                  limitations. Therefore, consistent labeling will help. 
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ウェブページ一式で繰り返して表示される機能的なコンポーネントを一貫して識別できるようにすることである。ウェブサイトを操作する際にスクリーンリーダーを使用している利用者は、複数のウェブページで使われている機能に馴染みがあるかどうかに大きく依存している。全く同じ機能が、複数のウェブページ毎に異なるラベル (すなわち、より一般的には、異なる <a href="https://www.w3.org/TR/accname-aam-1.1/">アクセシブルな名前 (accessible name)</a>) を付けられている場合、そのウェブサイトはかなり使いづらいものになってしまう。それは、認知的限界のある人々に混乱をもたらし、認知的負荷を増大させてしまうことがある。したがって、一貫したラベルが役立つ。
                   
                </p>
-               <p>This consistency extends to the text alternatives. If icons or other non-text items
-                  have the same functionality, then their text alternatives should be consistent as
-                  well.
+               <p>この一貫性は、テキストによる代替にも及ぶ。アイコン又はその他の非テキスト項目が同じ機能を持つ場合、そのテキストによる代替も同様に一貫性をもたせることが望ましい。
                   
                </p>
-               <p>If there are two components on a web page that both have the same functionality as
-                  a component on another page in a set of web pages, then all 3 must be consistent.
-                  Hence the two on the same page will be consistent.
+               <p>ウェブページ一式の中で、もしウェブページに二つコンポーネントがあり、そのどちらも別のページにあるコンポーネントと同じ機能を持っているのであれば、三つのコンポーネントは一貫していなければならない。よって、同じページにある二つのコンポーネントは一貫していることになる。
```

Comment:
読んでいて何が「三つ」なのかわからなくなり混乱します。原文だと two components と a component で 3つとわかります。 a component の a を「一つの」と訳出してはどうでしょうか。

```suggestion
               <p>ウェブページ一式の中で、もしウェブページに二つコンポーネントがあり、そのどちらも別のページにある一つのコンポーネントと同じ機能を持っているのであれば、三つのコンポーネントは一貫していなければならない。よって、同じページにある二つのコンポーネントは一貫していることになる。
```